src/build/intro-currency-runtime.js

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
Generating only the header should be pretty trivial, don't you think? The rest of the files obviously would be much harder to generate...